src/quickpicks/uris.ts

```diff
@@ -88,6 +88,7 @@ export async function uriQuickpick(
   });
 
   const quickPick = window.createQuickPick();
+  quickPick.ignoreFocusOut = true;
```

Comment:
Unrelated to this PR, but was helpful to enable for test debugging and is generally nice to set when in a multi-step quickpick flow.